src/pages/home/ReportScreen.js

```diff
@@ -58,6 +59,8 @@ const propTypes = {
 
     /** Beta features list */
     betas: PropTypes.arrayOf(PropTypes.string),
+
+    isLoadingReportData: PropTypes.bool,
```

Comment:
Comment.... :wheel_of_dharma:

Reply:
What's the ☸️ emoji for @parasharrajat? Just curious.

Reply:
Just for pun